alphabet.py

```diff
@@ -0,0 +1,9 @@
+def get_letters():
+    """
+    :return: A list containing all the capital and the lower alphabet.
+    """
+    return [chr(char_unicode) for char_unicode in range(ord("A"), ord("z") + 1) if chr(char_unicode).isalpha()]
```

Comment:
This can be done more eficciently, with out checking if each character is a letter.